src/v/config/node_config.cc

```diff
@@ -169,6 +172,46 @@ node_config::node_config() noexcept
       "injection",
       {.visibility = visibility::tunable},
       std::nullopt)
+  , debug_bundle_write_dir(
+      *this,
+      "debug_bundle_write_dir",
+      "The full path to the directory where debug bundle zip files are stored "
+      "(default: "
+      "/tmp)",
+      {.visibility = visibility::user},
+      "/tmp",
+      [](const std::filesystem::path& write_dir) -> std::optional<ss::sstring> {
+          if (!std::filesystem::exists(write_dir)) {
+              return ss::sstring{"Debug bundle write dir does not exist"};
+          }
+
+          // access is apart of unistd.h
+          if (access(write_dir.c_str(), R_OK | W_OK) != 0) {
+              return ss::sstring{"Debug bundle write dir does not have read "
+                                 "and write permissions"};
+          }
```

Comment:
i would be concerned that these calls might cause a reactor stall. you could add them some where like `application::check_environment`?

Reply:
We sync'd offline about this. The plan is to do validation in application.cc